spot-ml/ml_test.sh

```diff
@@ -70,4 +71,8 @@ time spark-submit --class "org.apache.spot.SuspiciousConnects" \
   --scored ${HDFS_SCORED_CONNECTS} \
   --threshold ${TOL} \
   --maxresults ${MAXRESULTS} \
-  --ldamaxiterations 11
\ No newline at end of file
+  --ldamaxiterations 11 \
```

Comment:
this is the ml_test.sh script but why are we using a different value than what is in spot.conf?